src/consts/index.ts

```diff
@@ -26,4 +26,6 @@ export const GCS_CLIENT_EMAIL = process.env.GCS_CLIENT_EMAIL || '';
 export const GCS_PRIVATE_KEY = process.env.GCS_PRIVATE_KEY || '';
 export const GCS_BASE_URL = 'https://storage.googleapis.com';
 
+export const GOOGLE_MAPS_API_KEY = process.env.GOOGLE_MAPS_API_KEY || '';
```

Comment:
Remember to add to Heroku the env variable

Reply:
ok done, also added to github secrets